main.go

```diff
@@ -114,20 +115,33 @@ func main() {
 			if (*flagMaxFails > 0) && (fails >= *flagMaxFails) {
 				atomic.StoreInt32(&exitFlag, 1)
 			}
-			f, err := ioutil.TempFile("", "go-stress")
-			if err != nil {
-				fmt.Printf("failed to create temp file: %v\n", err)
-				os.Exit(1)
-			}
-			f.Write(out)
-			f.Close()
-			if len(out) > 2<<10 {
-				out = out[:2<<10]
+			if *flagStdErr {
+				fmt.Fprintf(os.Stderr, "\n%s\n", out)
+				if len(out) > 2<<10 {
+					out = out[:2<<10]
+				}
+				fmt.Printf("\n%s\n", out)
```

Comment:
Easy enough.  I just removed the stdout lines about the failure.  It was spamming my screen.  The ticker will show the number of failures.